static/src/stylesheets/module/signup/_newsletters.scss

```diff
@@ -0,0 +1,151 @@
+.newsletters-heading {
+    @include fs-header(5);
+    padding: $gs-baseline 0;
+
+    @include mq($from: tablet) {
+        font-size: 36px;
+        margin-top: 8px;
+  }
+}
+
+.newsletter-card__signup {
+    padding: .5rem;
+}
+
+.newsletter-card__example {
+    display: inline;
+    float: right;
+
+    .newsletter-card__lozenge {
+        color: $news-main-1;
+        border-color: $news-main-1;
+        -webkit-border-radius: 62.5rem;
+        border-radius: 100px;
+        height: 2rem;
+        width: 6rem;
+        margin: 0;
+
+        &:hover,
+        &:focus,
+        &:active {
+            border-color: lighten($news-main-1, 10%);
+        }
+    }
+}
+
+.newsletter-card__wrapper {
+    position: relative;
+
+    @include mq(desktop) {
+        margin-left: 17%;
+    }
+}
+
+.newsletter-card {
+    color: $neutral-1;
+    display: inline-block;
+    float: left;
+    height: 16.5rem;
```

Comment:
I think having all these fixed heights may come back to bite someone since if the text overflows it will look bad - perhaps `min-height` might be more appropriate? Or even better, using flex-box like we do on fronts so that the container can expand as necessary but all columns will have the height of the tallest - I think it's this technique: https://css-tricks.com/fluid-width-equal-height-columns/